### ISSUE TITLE
Project Complete?

### ISSUE THREAD

**Issue by zannain** [Jaadawg/jamming#5]
Hey Jayden,

Can you confirm this is your final project? When I go to compile your project and run a search I'm getting an `Illegal redirect uri`.

![image](https://user-images.githubusercontent.com/24375741/36189133-8e1ed7da-111e-11e8-81d9-b6a849c5a84a.png)

**Comment by Jaadawg** [comment]
It is! I had it working I thought. Do you know why that error keeps coming
up? I'm not sure how to solve that problem.

**Comment by zannain** [comment]
You may have changes you need to push up to your repository. Heres the list of commands you should try to execute in your terminal when you are in your project directory to do that:

1. `git status` - *Check if you have anything that needs to be staged, (look for any files in RED). If it says something like "nothing to commit" I would recommend contacting an advisor to help you work through this issue and once you have everything working go to step 2. Otherwise, go straight to step 2 if you do see files in red.*
2. `git add .` - *This will stage all files you have made changes to since your most recent commit*
3. `git commit -m "message describing commit"` 
4. `git push origin master`

Once you've completed these steps let me know!

**Comment by Jaadawg** [comment]
Yep I've done all of that already. That's why I figured the project was
good to push! I am not sure what could be the matter at this point. I will
go back on and try to get some help.

**Comment by Jaadawg** [comment]
Okay so when you clone the project and run it locally it works perfectly.
There was just issues with the redirect, aka a spelling mistake in my
spotify dashboard. Anyways when you run it locally it works very well.
Unfortunately working with Carolina, we couldn't figure out why running
from surge directly won't work properly. So just clone the project and run
it from localhost and it works great! Thanks.
